### ISSUE TITLE
moveToSnapshots logs an upgrade to a version that does not exist yet.

### ISSUE THREAD

**Issue by snicoll** [spring-projects/spring-boot#35296]
When `moveToSnapshots` is used, it logs the same thing than an actual upgraded, e.g.

```
Upgrade to Spring Framework 5.3.28
```

However, that's not really what it has done. While it has created an issue with that title, it really switched the build to use `5.3.28-SNAPSHOT`. I wonder if the log could be rephrased, something like:

```
Start building against Spring Framework 5.3.28 snapshots
```

**Comment by wilkinsona** [comment]
I'm not sure what happened here as it should already do that:

https://github.com/spring-projects/spring-boot/blob/56af1569d3edde34d12ff9ba49b5810bca544f1e/buildSrc/src/main/java/org/springframework/boot/build/bom/bomr/MoveToSnapshots.java#L49-L53

[This commit](https://github.com/spring-projects/spring-boot/commit/5fdd124bc3bf9917067ab328a26dd0bb5047c6d0) was generated by Bomr and the commit message looks good to me.

**Comment by snicoll** [comment]
The commit message is not what I am talking about here. I mean the log statement in the console.

**Comment by snicoll** [comment]
> The current logging is describing the issues that have been opened.

Yes, but in the context of what it actually does, I find that misleading. I don't know what suffix could help with that though.
